javascript/GameBoard.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-use-before-define */
+/* global player, checkWinning, checkTie */
+const GameBoard = (() => {
+  let board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '];
+
+  const player1 = player('player1', 'X');
+  const player2 = player('player2', 'O');
+  let gameStatus = 'play';
+  let Winner;
```

Comment:
Please, respect the nomenclature of javascript variable. They have to start with lower character.